plugins/BEdita/Core/src/Model/Entity/Property.php

```diff
@@ -210,6 +210,9 @@ public function getSchema($accessMode = null)
             // Booleans are valid schemas, though they're quite uncommon.
             return $schema;
         }
+        if (empty($schema)) {
+            $schema = new \stdClass();
```

Comment:
What happens if `$schema = new \stdClass();` and is not nullable?
At row `230` there is an array assignement `$schema['$id'] = sprintf('/properties/%s', $this->name);` that would give an error as `Cannot use object of type stdClass as array.`

Reply:
Right, in case of empty array as `$schema` we should probably have just 

```null
  $schema = [
                'oneOf' => [
                    [
                        'type' => 'null',
                    ],
                    new \stdClass(),
                ],
   ];
```

When reading a property schema if there's no `type` directive we should treat the property as JSON